server/src/main/java/org/opensearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -1059,6 +1077,42 @@ public void onFailure(Exception ex) {
         }
     }
 
+    private void releaseResourceLockFiles (
+        Collection<SnapshotId> snapshotIds,
+        RepositoryData repositoryData,
+        RemoteStoreLockManagerFactory remoteStoreLockManagerFactory,
+        ActionListener<Void> onAllShardsCompleted
+    ){
+        for (SnapshotId snapshotId: snapshotIds) {
+            List<String> indices = this.getSnapshotInfo(snapshotId).indices();
+            List<IndexId> indexIds = repositoryData.resolveIndices(indices);
+            for (IndexId indexId: indexIds) {
+                try {
+                    IndexMetadata indexMetadata = this.getSnapshotIndexMetaData(repositoryData,
+                        snapshotId, indexId);
+                    if (this.getSnapshotInfo(snapshotId).isRemoteStoreInteropEnabled()
```

Comment:
this should be `isRemoteStoreIndexShallowCopyEnabled` ?

Reply:
Updated in new revision